db/migrate/20191118200509_add_user_and_repo_to_project.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserAndRepoToProject < ActiveRecord::Migration[6.0]
+  def change
+    add_column :projects, :user, :string
```

Comment:
Maybe `user_repo`, wdyt? It might be confusing to have this field, called user, inside project table.